packaging/linux/snap/snapcraft.yaml

```diff
@@ -0,0 +1,24 @@
+name: keybase
+version: master
+summary: Crypto for everyone!
+description: |
+  Keybase service and command line client.
+
+grade: devel # must be 'stable' to release into candidate/stable channels
+confinement: strict
+
+apps:
+  keybase:
+    command: keybase
+    plugs: [network, network-bind]
+
+parts:
+  client-go:
+    source: ../../go
+    plugin: go
+    go-importpath: github.com/keybase/client/go
+    go-packages: [github.com/keybase/client/go/keybase]
+    go-buildtags: [production]
+    after: [go]
+  go:
+    source-tag: go1.7.5
```

Comment:
I'm a big fan of generating files: https://github.com/keybase/client/blob/master/packaging/linux/deb/package_binaries.sh#L59-L65